packages/astro/src/types/public/config.ts

```diff
@@ -563,14 +562,14 @@ export interface AstroUserConfig {
 		 * @type {string}
 		 * @default `'./dist/client'`
 		 * @description
-		 * Controls the output directory of your client-side CSS and JavaScript when `output: 'server'` or `output: 'hybrid'` only.
+		 * Controls the output directory of your client-side CSS and JavaScript when building a website with server-rendered pages.
 		 * `outDir` controls where the code is built to.
 		 *
 		 * This value is relative to the `outDir`.
 		 *
 		 * ```js
 		 * {
-		 *   output: 'server', // or 'hybrid'
+		 *   output: 'server',
 		 *   build: {
 		 *     client: './client'
 		 *   }
```

Comment:
lower in this file, the JSDoc for server islands still mention output hybrid. I didn't update it since once it's unflagged the jsdoc won't exist anymore.